docs/book/src/developer/providers/v1.3-to-v1.4.md

```diff
@@ -19,7 +19,9 @@ maintainers of providers and consumers of our Go API.
 
 ### Deprecation
 
--
+- The api versions `v1alpha3` and `v1alpha4` are deprecated and will be removed. 
```

Comment:
Good catch! Could you maybe add a link to the mention of this in the contributors guide? Just so people know where to go for more information. i.e.   https://release-1-4.cluster-api.sigs.k8s.io/contributing?highlight=v1alpha#removal-of-v1alpha3--v1alpha4-apiversions

Reply:
Great addition. Added.